modules/local/merge_reports/main.nf

```diff
@@ -0,0 +1,32 @@
+process MERGE_REPORTS {
+    tag "$benchmark_tool"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-2076f4a3fb468a04063c9e6b7747a630abb457f6:fccb0c41a243c639e11dd1be7b74f563e624fcca-0':
+        'biocontainers/mulled-v2-2076f4a3fb468a04063c9e6b7747a630abb457f6:fccb0c41a243c639e11dd1be7b74f563e624fcca-0' }"
+
+    input:
+    tuple val(benchmark_tool), path(inputs)
```

Comment:
I think you should keep `meta` here and keep the benchmark tool inside of the meta. That way you don't lose your other data in case you ever want to anything with it

Reply:
that makes sense :D